pkg/controller/account/account_controller.go

```diff
@@ -9,10 +9,6 @@ import (
 	"strings"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/awserr"
-	"github.com/aws/aws-sdk-go/service/iam"
-	"github.com/aws/aws-sdk-go/service/organizations"
 	"github.com/aws/aws-sdk-go/service/sts"
 	"github.com/go-logr/logr"
 	"github.com/openshift/aws-account-operator/pkg/controller/utils"
```

Comment:
I also noticed in this file (because I had to add it to the other file as well) we're referencing the controller utils twice, once as `utils` here and again as `controllerutils` in the expanded view below.  Should I fix that in this PR or make another card for that?

Reply:
I would say consolidate them in this PR